library/src/webapp/js/portal/portal.init.js

```diff
@@ -21,7 +21,8 @@ document.addEventListener("DOMContentLoaded", () => {
   });
 
   searchPanel?.addEventListener("hidden.bs.offcanvas", e => {
-    e.target.querySelector("sakai-search")?.clear();
+    const input = e.target.querySelector("#sakai-search-input");
```

Comment:
I would not be doing this. Add a clear method to sakai-search.js instead. We should not be addressing elements inside a component's markup if we can avoid it. This means that if we ever want to move to shadow dom, this stuff will need less updating.